strr-web/components/bcros/form-section/property/Details.vue

```diff
@@ -104,25 +149,56 @@ const ownershipType = defineModel<string>('ownershipType')
 const businessLicense = defineModel<string>('businessLicense')
 const parcelIdentifier = defineModel<string>('parcelIdentifier')
 const businessLicenseExpiryDate = defineModel<string>('businessLicenseExpiryDate')
+const typeOfSpace = defineModel<string>('typeOfSpace')
+const isOnSameProperty = defineModel<string>('isOnSameProperty')
+const numberOfRooms = ref<number>(1)
 
 watch(businessLicense, (): void => {
   if (!businessLicense.value) {
-    // clear exp date when business lic is empty
     formState.propertyDetails.businessLicenseExpiryDate = ''
   }
 })
 
-const emit = defineEmits(['validateOwnership', 'validateProperty', 'validateBusinessLicenseExpiryDate'])
+const handleIsOnSamePropertyChange = () => {
+  if (isOnSameProperty.value === 'No') {
```

Comment:
A little issue here, you are using translations to create an array, but here only check against English 'Yes' 'No'. It will not be a problem now, but if using anything else other then Yes/No might break it.

Also could be simplified: `numberOfRooms.value = isOnSameProperty.value === 'Yes' ? 1 : 0`